CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add `Config` CR resource management into the `release` controller.
 
+### Changed
+
+- Skip `App` CR resource creation if corresponding `Config` CR has no expected status.
+
```

Comment:
We don't have to mention that as the support for the Config CR is added within the release.

```suggestion
```